basic/ftp/src/test/resources/user.properties

```diff
@@ -1,4 +1,4 @@
 userid=demo
 password=demo
-port=3333
+availableServerPort=3333
```

Comment:
what's the difference here? how is this connected with the changes you did in the TestSuite class?